### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by VatsalPatel19** [ahmadfaizalbh/Chatbot#68] [Q&A] [answered]
can you please expalin yor project in simpler terms, and a breif introduction about how this project works.

**Comment by ahmadfaizalbh** [comment, marked as answer]
This is a chat bot framework/engine that you could use in your project to build chatbot.
This is a template based bot engine where intend match and entity extraction is done using regex.
Since its pure python template based bot framework  it's easy to build, integrate and customise functionality. 

In future versions we have planed to add more deep learning algorithms for intend classification & entity extraction.


Django wrapper module: https://github.com/ahmadfaizalbh/Django-Chatbot
Sample web chatbot using the above: https://github.com/ahmadfaizalbh/webbot
